### DISCUSSION TITLE
Does i18next help with Gendered Grammar?

### DISCUSSION THREAD

**Discussion by gracepotma** [i18next/i18next#2161] [Q&A]
Dear i18next community,
Our open-source application community loves using i18next for our JS app. 
We are just wondering, is there anything in the i18next framework that helps with Gendered Grammar, for applications that are _not_ storing and using the user's gender? 

Using the user's gender was the only use case we found in the docs (here: https://www.i18next.com/translation-function/context). 

**Are there any useful plugins or otherwise that are helpful for gendered translations in software that does not store the user's gender?** 

We're running into this question for languages like French (any many others) which require masculine/feminine conjugations for correct grammar. FWIW - I’ve looked through the web for guidance and the recommendations vary:

- Use the user’s gender ([example](https://www.linkedin.com/pulse/gender-translation-user-interfaces-eric-negron/#:~:text=femaleGreatJobMessage%2C%20maleGreatJobMessage%2C%20neutralGreatJobMessage)) - this just adds more complexity for us to maintain in our code. Maybe not avoidable?
- Re-phrase in a grammatically gender-neutral way: E.g. instead of Êtes-vous sûr de vouloir supprimer cette visite? we could use Voulez-vous vraiment annuler cet visite?
- Use () if neutral phrasing is not possible: E.g. "Êtes-vous sûr(e) de vouloir indiquer ce(tte) patient(e) comme décédé(e)"
- Use / as a last resort if () is not possible: E.g. "sibling" in French = "Soeur/Frère"

**Comment by jamuhl** [comment]
I'm not aware of any other solution than using the users gender to form a context variant (i18next) or a select (messageformat).
